fakeDataGenerator.js

```diff
@@ -0,0 +1,169 @@
+const faker = require('faker');
+const fs = require('fs');
+const moment = require('moment');
+
+faker.seed(105);
+
+const fakeRestaurants = [];
+
+for (let i = 1; i <= 100; i++) {
+  const fakeRestaurant = {id: i};
+  const fakeWords = [];
+  const length = faker.random.number() % 3;
```

Comment:
I think faker.random.number(3) will choose a random number between 1 and 3.

Reply:
Yeah, I mostly refactored after realizing that faker.random.number can take params. Missed that one, will do. Thanks!